corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -25,6 +29,12 @@ hqDefine("cloudcare/js/formplayer/menus/collections", [
         });
     }
 
+    function enableLanguageOption(response, collection) {
+        response.langs = response.locales.slice(1);
+        _.extend(collection, _.pick(response, collection.formProperties));
```

Comment:
What's this line for?

Reply:
lines 33 and 34 ensures that this collection obj (the menu obj) has a `langs` attribute, which is later used to provide the available language options in the hamburger menu